avanza/avanza.py

```diff
@@ -1705,3 +1715,108 @@ def create_monthly_saving(
                 }
             }
         )
+
+    def get_transactions(self,
+                         account_or_transaction_type: str,
+                         transactions_from: date,
+                         transactions_to: date,
+                         min_amount: int=None,
+                         max_amount: int=None,
+                         order_book_ids: Sequence[str]=[]):
+        """ Get transactions
+
+        Args:
+            account_or_transaction_type: A valid account ID or a transaction type.
+
+            transactions_from: Fetch transactions from this date.
+
+            transactions_to: Fetch transactions to this date.
+
+            min_amount: Only fetch transactions of at most this value.
+
+            max_amount: Only fetch transactions of at least this value.
+
```

Comment:
Shouldn't these me switched?

min_amount = at least
max_amount = at most

Reply:
You're right!